rdr_service/api/nph_participant_api_schemas/schema.py

```diff
@@ -50,6 +50,28 @@ def sort(context, sort_info, value):
         raise ValueError(f"{value} : Invalid Key -- Event Object Type")
 
 
+class GraphQLConsentEvent(ObjectType):
+    """ NPH ConsentEvent """
+
+    value = SortableField(
+        NonNull(String), sort_modifier=lambda context: context.set_order_expression(context.sort_table.status)
+    )
+    time = SortableField(
+        DateTime,
+        sort_modifier=lambda context: context.set_order_expression(context.sort_table.time)  # Order by time
+    )
+
+    opt_in = Field(NonNull(String))
+
+    @staticmethod
```

Comment:
Since we are copying this from another method and it has the same fn signature, can we make a mixin class  and utilize it thru inheritance?

Reply:
We might be able to do something even better here. `GraphQLConsentEvent` is the same as `Event`, just with adding in the `opt_in` field, right? What if instead we subclass the `Event` class rather than `ObjectType`?

Reply:
@jmtrujillo75, what do you think?

Reply:
@kenny-skaggs That should work. I tried that and it does work